### ISSUE TITLE
Seedlet: Console + accessibility focus error when using mobile navigation

### ISSUE THREAD

**Issue by kjellr** [Automattic/themes#3331]
This was reported [in the theme review for Blank Canvas](https://themes.trac.wordpress.org/ticket/94482#comment:14), but it's about Seedlet's menu:

> When I add a menu to the primary navigation and test the responsive version, 
there is a JS error, but this is from the parent theme:

```
primary-navigation.js?ver=1.2.4:42 Uncaught DOMException: 
Failed to execute 'querySelector' on 'Document': '.${ id }-navigation' is not a valid selector
```

> When I tab through the menu, the focus is not kept inside the modal, instead, when I tab away from 
the last menu item, the focus seems to be on the content below the menu.
> I can't tell where I am on the page because the open menu is covering the content.
> Perhaps this is because of the JS error.
